nifi-extension-bundles/nifi-snowflake-bundle/nifi-snowflake-services/src/main/java/org/apache/nifi/snowflake/service/SnowflakeComputingConnectionPool.java

```diff
@@ -142,6 +152,8 @@ public class SnowflakeComputingConnectionPool extends AbstractDBCPConnectionPool
             .expressionLanguageSupported(ExpressionLanguageScope.ENVIRONMENT)
             .build();
 
+    private volatile OAuth2AccessTokenProvider oauthService;
```

Comment:
Recommend adjusting the variable name to align with the class:
```suggestion
    private volatile OAuth2AccessTokenProvider accessTokenProvider;
```